### PR TITLE
Added Myself (Yash Verma)

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Google Codein Task

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://YashKumarVerma.github.io/gci15.fossasia.org

**Comment by YashKumarVerma** [comment]
Can someone help me ? I am not able to see the forked repository (this one on my account) as github pages. Please help !

**Comment by maitreyav** [comment]
Hi @YashKumarVerma, I also had the same issue. Refer [this.](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md)

File: _data/students.yml [hunk 1]
```diff
@@ -341,3 +341,11 @@
   blog: http://poprosturonin.com
   twitter: poprosturonin
   bio: Gamer, developer, designer from Poland.
+  
+- name: Yash Kumar Verma
+  github: YashKumarVerma
+  image: yash_kumar_verma.JPG
+  facebook: http://www.facebook.com/yashkumarverma
+  twitter: @yash_kr_verma
```
Comment (roonyh) [inline _data/students.yml]: Don't add the `@`

**Comment by roonyh** [comment]
@YashKumarVerma Its because the page has errors. Remove the `@` mark and it should be fine.

**Comment by YashKumarVerma** [comment]
- Added Myself
- Added Thanks for Contributing
- Made a table around Loklak peers
- Added My Project To Gallery
- _Fixed Improper (") in thanks for contributing script.

please merge as soon as possible.

Thanks in advance.

**Comment by jigyasa-grover** [comment]
@YashKumarVerma Resolve merge conflicts.

**Comment by YashKumarVerma** [comment]
um... may you please tell me. I am a beginner here

**Comment by YashKumarVerma** [comment]
ok. I am closing this pull request. I will do the entire process again. Thanks for help guys.

**Pull request closed without merging**